packages/block-library/src/site-title/index.php

```diff
@@ -24,20 +24,20 @@ function render_block_core_site_title( $attributes ) {
 	$aria_current = is_home() || ( is_front_page() && 'page' === get_option( 'show_on_front' ) ) ? ' aria-current="page"' : '';
 
 	if ( isset( $attributes['level'] ) ) {
-		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . $attributes['level'];
+		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . (int) $attributes['level'];
 	}
 
 	if ( $attributes['isLink'] ) {
 		$link_attrs = array(
-			'href="' . get_bloginfo( 'url' ) . '"',
-			'rel="home"',
+			'href="' . esc_url( get_bloginfo( 'url' ) ) . '"',
+			esc_attr( 'rel="home"' ),
```

Comment:
`esc_attr` won't work here... it can be used in the values but not for the key-value pair. Besides that however, since this is a hardcoded value it is unnecessary.

Reply:
Yes you are correct. Let me adjust this.

My point above about being over zealous about escaping everything also applies here.

Reply:
I've escaped the value. This one might one day changed to be configurable as a block attribute so better safe than sorry. Minor overhead.